package.json

```diff
@@ -19,7 +19,11 @@
     }
   },
   "lint-staged": {
-    "*.{js,jsx}": "./node_modules/.bin/eslint --ext .js,.jsx src/ && cross-env jest --collect-coverage --env=jsdom"
+    "*.{js,jsx}": [
+      "npm run lint:fix",
```

Comment:
npm run lint isn’t required if you use npm run lint:fix

Reply:
Done